internal/gazelle.bash.in

```diff
@@ -32,7 +32,7 @@ function find_runfile {
     readlink "$runfile"
     return
   fi
-  runfile=$(echo "$runfile" | sed -e 's!^\(\.\./\|external/\)!!')
+  runfile=${runfile#external/}
```

Comment:
It may be important to note that this does not trim `../`. This is because `--experimental_sibling_repository_layout` will fast path to the above `readlink` and is therefore not needed.